config/initializers/queries.rb

```diff
@@ -2,19 +2,18 @@ def transliterate(name)
   # remove diacritics from name
   ActiveSupport::Inflector.transliterate(name)
 end
-# standard api query for all Contemporary Poets' names
+# standard api query for all Index of Poets names
 sort_by = API_OPTS["browse_sort"]
 options = {
   facet: "person.name",
   facet_num: 1000,
   facet_sort: sort_by,
   num: 0,
-  f: ["subcategory|Contemporary Poet"]
```

Comment:
You went from plural to singular here - I think it's worth thinking about this just a bit. What are the other subcategories used? are they singular?

Reply:
This was discussed on Teams and I believe the decision was to keep it singular.